packages/lib/modules/lbp/steps/useCreateLbpStep.tsx

```diff
@@ -77,36 +88,36 @@ export function useCreateLbpStep(): TransactionStep {
   const blockProjectTokenSwapsIn = userActions === 'buy_only' ? true : false
 
   const { symbol: launchTokenSymbol } = useTokenMetadata(launchTokenAddress, selectedChain)
-  const { symbol: collateralTokenSymbol } = useTokenMetadata(collateralTokenAddress, selectedChain)
+  const { symbol: reserveTokenSymbol } = useTokenMetadata(reserveTokenAddress, selectedChain)
 
   const hasRequiredValues =
-    launchTokenSymbol &&
-    collateralTokenSymbol &&
-    name &&
-    chainId &&
-    userAddress &&
-    launchTokenAddress &&
-    collateralTokenAddress &&
-    projectTokenStartWeight &&
-    reserveTokenStartWeight &&
-    projectTokenEndWeight &&
-    reserveTokenEndWeight &&
-    startTime &&
-    endTime
+    !!launchTokenSymbol &&
+    !!reserveTokenSymbol &&
+    !!name &&
+    !!chainId &&
+    !!userAddress &&
+    !!launchTokenAddress &&
+    !!reserveTokenAddress &&
+    !!projectTokenStartWeight &&
+    !!reserveTokenStartWeight &&
+    !!projectTokenEndWeight &&
+    !!reserveTokenEndWeight &&
+    !!startTime &&
+    !!endTime
 
   const createPoolInput = hasRequiredValues
     ? {
         protocolVersion: 3 as const,
         poolType: PoolType.LiquidityBootstrapping,
-        symbol: `${launchTokenSymbol}-${collateralTokenSymbol}-LBP`,
+        symbol: `${launchTokenSymbol}-${reserveTokenSymbol}-LBP`,
         name: `${name} Liquidity Bootstrapping Pool`,
-        swapFeePercentage: parseUnits('0.01', 18), // TODO: confirm default value
+        swapFeePercentage: parseUnits('0.01', 18), // TODO: confirm default value?
```

Comment:
did we ever get final decision on default swap fee % ?

Reply:
I think there is going to be some UI to be able to customize it.

Reply:
<img width="398" alt="Screenshot 2025-06-30 at 15 15 12" src="https://github.com/user-attachments/assets/03aa01ca-a698-49ba-8552-416490edb7ef" />

If it's 1% we will have to change the preview though